### PR TITLE
[zh] resync page kubelet-config.v1beta1

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
This PR resyncs page `kubelet-config.v1beta1`.

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 1]
```diff
@@ -1878,6 +1901,32 @@ Default: true -->
...
 </td>
 </tr>
+<tr><td><code>containerRuntimeEndpoint</code> <B>[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--ContainerRuntimeEndpoint is the endpoint of container runtime.
+Unix Domain Sockets are supported on Linux, while npipe and tcp endpoints are supported on Windows.
+Examples:'unix:///path/to/runtime.sock', 'npipe:////./pipe/runtime'-->
+   <p>ContainerRuntimeEndpoint 是容器运行时的端点。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md:1911]:
```suggestion
   <p>containerRuntimeEndpoint 是容器运行时的端点。
```

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 2]
```diff
@@ -1878,6 +1901,32 @@ Default: true -->
...
+Linux 支持 Unix 域套接字，而 Windows 支持 npipe 和 tcp 端点。
+示例：'unix://path/to/runtime.sock', 'npipe:////./pipe/runtime'</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md:1913]:
```suggestion
示例：'unix://path/to/runtime.sock', 'npipe:////./pipe/runtime'。</p>
```

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 3]
```diff
@@ -1878,6 +1901,32 @@ Default: true -->
...
+</tr>
+<tr><td><code>imageServiceEndpoint</code><br/>
+<code>string</code>
+</td>
+<td>
+   <!--ImageServiceEndpoint is the endpoint of container image service.
+Unix Domain Socket are supported on Linux, while npipe and tcp endpoints are supported on Windows.
+Examples:'unix:///path/to/runtime.sock', 'npipe:////./pipe/runtime'.
+If not specified, the value in containerRuntimeEndpoint is used.-->
+   <p>ImageServiceEndpoint 是容器镜像服务的端点。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md:1924]:
```suggestion
   <p>imageServiceEndpoint 是容器镜像服务的端点。
```

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md [hunk 4]
```diff
@@ -1878,6 +1901,32 @@ Default: true -->
...
+Linux 支持 Unix 域套接字，而 Windows 支持 npipe 和 tcp 端点。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1beta1.md:1925]:
```suggestion
Linux 支持 UNIX 域套接字，而 Windows 支持命名管道和 TCP 端点。
```

**Comment by windsonsea** [comment]
/lgtm

please run the following command to reset the ci:

```
git reset
git commit --amend --no-edit
git push -f
```

**Pull request merged**